<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMPSettingsGeneral</class>
 <widget class="QWidget" name="PageMPSettingsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldNewRoundResetsPlayers">
     <property name="text">
      <string>Reset Players on New Round</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldPerfectionEnabled">
     <property name="toolTip">
      <string>Controls whether the player can earn the Perfection medal in this gametype.</string>
     </property>
     <property name="text">
      <string>Perfection Medal Enabled</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldTeamsEnabled">
     <property name="text">
      <string>Teams Enabled</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelRoundLimit">
     <property name="text">
      <string>Number of Rounds</string>
     </property>
     <property name="buddy">
      <cstring>fieldRoundLimit</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelRoundsToWin">
     <property name="text">
      <string>Rounds to Win</string>
     </property>
     <property name="buddy">
      <cstring>fieldRoundsToWin</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelRoundTimeLimit">
     <property name="text">
      <string>Round Time Limit</string>
     </property>
     <property name="buddy">
      <cstring>fieldRoundTimeLimit</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="fieldGracePeriod">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="fieldRoundTimeLimit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>no limit</string>
     </property>
     <property name="suffix">
      <string> min</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldNewRoundResetsMap">
     <property name="text">
      <string>Reset Map on New Round</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="fieldRoundsToWin">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="fieldSuddenDeathTime">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelGracePeriod">
     <property name="text">
      <string>Grace Period</string>
     </property>
     <property name="buddy">
      <cstring>fieldGracePeriod</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelSuddenDeath">
     <property name="text">
      <string>Sudden Death Time</string>
     </property>
     <property name="buddy">
      <cstring>fieldSuddenDeathTime</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="fieldRoundLimit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>no limit</string>
     </property>
     <property name="suffix">
      <string> rounds</string>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelScoreToWin">
     <property name="text">
      <string>Score to Win</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="fieldScoreToWin">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>no limit</string>
     </property>
     <property name="suffix">
      <string> points</string>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
